browser/android/brave_cosmetic_resources_tab_helper.cc

```diff
@@ -24,6 +24,109 @@
 
 
 namespace {
+const char k_observing_script[] =
```

Comment:
I think we'll probably want a `transpile_web_ui` target for just `content_cosmetic.ts` and put the result in a grd file to isolate it from everything else in `brave_extension`

Reply:
it's done via transpile now